### ISSUE TITLE
GUI command not working in 9.16.0 and later

### ISSUE THREAD

**Issue by Hypnosphi** [gemini-testing/html-reporter#531]
Looks like it treats "gui" as a test path instead

```
% npx hermione gui
[20:22:09 GMT+1] Error: Cannot find files by specified paths: gui
```

It works OK in 9.15.0

**Comment by DudaGod** [comment]
We will fix it today in this PR - https://github.com/gemini-testing/html-reporter/pull/530 and release new version.
Thank you.

**Comment by sipayRT** [comment]
fixed in html-reporter@9.16.3
